README.md

```diff
@@ -353,9 +353,9 @@ For example:
 
 ## Environment Variables
 
-If you wish to pass through environment variables through to the Docker container that powers this action you need to prefix the environment variable with `RENOVATE_`.
+To pass through environment variables through to the Docker container that powers this action, you must prefix the environment variable with `RENOVATE_`.
```

Comment:
🤔 We should change that, as these variables can interfere with renovate config options 🤔 . Better to use a custom prefix and add it to the `env-regex`.

Reply:
yes

Reply:
I'll let someone else fix up the readme, see issue:

https://github.com/renovatebot/github-action/issues/850